.travis.yml

```diff
@@ -7,7 +7,6 @@ node_js:
 - '6.1'
 env:
   matrix:
-  - TEST_BROWSER_NAME=PhantomJS
```

Comment:
`karma run --browser chrome` will run the entire test suite without doing much of anything beyond that (for instance).  Unfortunately, the maintainers of the PhatomJS NPM modules we were depending on haven't updated to Phantom 2.5 yet, so we'll have to wait.